jekyll/_cci2/server-3-operator-metrics-and-monitoring.adoc

```diff
@@ -72,96 +34,6 @@ log stream.
 https://prometheus.io/[Prometheus] is a leading monitoring and alerting system for Kubernetes. Server ships with basic
 implementation of monitoring common performance metrics.
 
-## Monitoring Metrics
```

Comment:
This information in "Monitoring Metrics" portion is not relevant right now but it will be once we add Telegraf back in. Not sure if you just want to comment this stuff out for now or leave a note letting the next person who works on this that they can find the info in an older commit.